### ISSUE TITLE
Accept --chain and --base-path from command line

### ISSUE THREAD

**Issue by gagarin55** [ETCDEVTeam/emerald-vault#146]
As a user I expect something like:
```
emerald account new --base-path=./ --chain=eth
```

**Comment by 652h** [comment]
@gagarin55 Can I try to fix it?

**Comment by 652h** [comment]
Just as a supplement to working `emerald --base-path=./ --chain=eth account new`?

**Comment by splix** [comment]
Yes, right. Currently it reads the path from env variable, but should be able to get as an option too

**Comment by 652h** [comment]
There is a support to take base path as a parameter, this variant works - `emerald --base-path=abc --chain=eth account new` [1].

[1] https://github.com/ETCDEVTeam/emerald-vault/blob/master/cli.yml#L16

**Comment by splix** [comment]
@gagarin55 ^^^

**Comment by 652h** [comment]
Environment variable is taken only if the option is not specified explicitly - https://github.com/ETCDEVTeam/emerald-vault/blob/master/src/cmd/mod.rs#L39

**Comment by splix** [comment]
AFAIK there was a problem in some of the cases when it didn't accept it from command line. @gagarin55 can provide more details imho

**Comment by 652h** [comment]
May be it was wrong sequence, for example `emerald --base-path=. ...` works, but `emerald ... --base-path=.` isnt't. It works only if options stay before command, not after. But anyway, I am waiting for more details.
